doc/crypto/api/keys/policy.rst

```diff
@@ -250,6 +261,31 @@ The usage flags are encoded in a bitmask, which has the type `psa_key_usage_t`.
 
     If this flag is present on all keys used in calls to `psa_key_derivation_input_key()` for a key derivation operation, then it permits calling `psa_key_derivation_verify_bytes()` or `psa_key_derivation_verify_key()` at the end of the operation.
 
+.. macro:: PSA_KEY_USAGE_ENCAPSULATE
+    :definition: ((psa_key_usage_t)0x00040000)
+
+    .. summary::
+        Permission to encapsulate new keys.
+
+    This flag is required to encapsulate new keys to send to a counter party.
```

Comment:
Key encapsulation is always about establishing a 'new' key. That said, this text is not consistent with the other usage flags. _If_ we keep this (see comment on L99), then it might be better worded as:
 > Permission to perform key encapsulation with the key.